docs/learn/learn-treasury.md

```diff
@@ -330,4 +330,4 @@ migration to a minority fork. However, the possibility of this scenario is quite
 
 - [Substrate's Treasury Pallet](https://github.com/paritytech/substrate/blob/master/frame/treasury/src/lib.rs) -
   The Rust implementation of the Treasury.
-  ([Docs](https://docs.substrate.io/rustdocs/latest/pallet_treasury/index.html))
+  ([Docs](https://paritytech.github.io/substrate/master/pallet_treasury/index.html))
```

Comment:
So, all the rustdocs now live on https://paritytech.github.io/substrate/ ?

Reply:
yes, specifically master. Others will be deprecated.

Reply:
hope to be resolved soon, I will make a PR here when I have a location for it